webpack.config.js

```diff
@@ -16,8 +16,8 @@ try {
     console.warn('\nWARNING! Failed to load DHIS config:', e.message);
     console.info('Using default config');
     dhisConfig = {
-        baseUrl: 'http://localhost:8090/',
-        authorization: 'Basic ZG92cmU6VGVzdDEyMyE=' //c2VsOlRlc3QxMjMh sel:Test123!    //bWFya3VzOlN0b3BDb3ZpZDE5IQ== markus:StopCovid19!    // YWRtaW46ZGlzdHJpY3Q=' // admin:district
+        baseUrl: 'http://localhost:8080/',
+        authorization: 'Basic YWRtaW46ZGlzdHJpY3Q=' //c2VsOlRlc3QxMjMh sel:Test123!    //bWFya3VzOlN0b3BDb3ZpZDE5IQ== markus:StopCovid19!    // YWRtaW46ZGlzdHJpY3Q=' // admin:district
```

Comment:
Was this config change supposed to be part of the commit?

Reply:
Ah, no :+1:

Reply:
What about the port change?
localhost:8080 > localhost:8090

Reply:
Ahem, yes, that's also a debug artifact...